.github/workflows/pull_request.yml

```diff
@@ -0,0 +1,23 @@
+name: CI
+
+on: pull_request
+
+jobs:
+  linux-unit-test:
+    runs-on: ubuntu-18.04
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v1
+    - name: Linux debug build
+      run: make TARGET=Linux all
+    - name: Linux release build
+      run: make TARGET=Linux BUILD_TYPE=Release all
+  linter:
```

Comment:
I don’t know how others feel about this, but I prefer to seperate jobs into different `.yml` files, with different `name` values.